certbot/error_handler.py

```diff
@@ -19,6 +19,14 @@
              signal.SIGXCPU, signal.SIGXFSZ])
 
 
+class SignalExit(Exception):
+    """This exception is used stop of execution of the code in the body of the
+    ErrorHandler context manager.
```

Comment:
nit: The docstring here should have the following form, according to [our style guide](https://certbot.eff.org/docs/contributing.html#coding-style) (and more specifically [pep8](https://www.python.org/dev/peps/pep-0008/) and [pep257](https://www.python.org/dev/peps/pep-0257/)):

```
"""one-line-docstring"""
```

or

```
"""one-line-doc-string

more-info

"""
```

Where each line is no more than 72 characters in length.